core/docker-entrypoint.sh

```diff
@@ -131,6 +132,9 @@ configuration: /etc/mailman-hyperkitty.cfg
 
 EOF
 
+# Generate a basic gunicorn.cfg.
+echo '[gunicorn]' > /etc/gunicorn.cfg
```

Comment:
Can you make sure that a basic gunicorn.cfg is same as the one included in Core by default because this will overwrite the existing configuration.

https://gitlab.com/mailman/mailman/-/blob/master/src/mailman/config/gunicorn.cfg